README.md

```diff
@@ -1,37 +1,155 @@
-# Tlf ham radio contest logger
+# TLF ham radio contest logger
 
 [![Build Status](https://github.com/Tlf/tlf/actions/workflows/ci-build.yml/badge.svg)](https://github.com/Tlf/tlf/actions/workflows/ci-build.yml)
 
 ## Summary
 
-Tlf is a console (ncurses) mode general purpose CW/VOICE keyer, logging and 
-contest program for hamradio. 
+TLF is a logging program for radiosport, or ham radio contests. It supports CQWW, CQ-WPX, ARRL-DX, ARRL-FD, PACC, and EU SPRINT contests as well as many more, and also has general QSO and DXpedition modes. Users can add new contests by editing a text file.
```

Comment:
`TLF` vs. `Tlf`: we shall use a unique naming in the docs and in the messages presented to the user.
I have no preference, but once we decide then we must stick to it.

Reply:
I agree. I went with `TLF` in my edits, but may have missed some instances.

Reply:
I also mean it generally: there are messages like "Do you want to leave Tlf?" in the code and also in other files (e.g. `NEWS`). So if we settle with `TLF` then those shall also be updated. Conversely, if `Tlf` is chosen then `TLF`s have to be changed.

Reply:
Please break all lines at no more than 80 characters.  Actually, anything in the range of 72 to 80 characters is fine.

Some file viewers do not wrap the lines so the user is left to be scrolling left and right, which is no fun at all!

Reply:
> `TLF` vs. `Tlf`: we shall use a unique naming in the docs and in the messages presented to the user.

I had a look into Rein's original webpage. It seems that the inconsistent use of mixing upper and lower cases has a long tradition :-). I found 'TLF' for the main naming, but in the various places also 'Tlf' and 'tlf'.  So maybe we should stay with the all uppercase version - 'TLF'.

We should decide now for one version and migrate documentation now and code step by step afterwards.

Reply:
Fine with me. So let's stick to TLF.

Reply:
> Please break all lines at no more than 80 characters. Actually, anything in the range of 72 to 80 characters is fine.

See https://github.com/dl1jbe/tlf/tree/improved_documentation